src/main/java/com/dumbdogdiner/stickyapi/bukkit/command/CooldownManager.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.command;
+
+import org.bukkit.command.CommandSender;
+
+import java.util.HashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class CooldownManager {
```

Comment:
Would be nice if this could be built for both Bukkit and Bungee? Perhaps using UUIDs rather than any form of player-related object.

Reply:
I'll see how bungee works, or make an abstract class